lib/client/api.go

```diff
@@ -852,6 +855,7 @@ func profileFromKey(key *Key, opts ProfileOptions) (*ProfileStatus, error) {
 		Logins:             sshCert.ValidPrincipals,
 		ValidUntil:         validUntil,
 		Extensions:         extensions,
+		CriticalOptions: 	sshCert.CriticalOptions,
```

Comment:
`gofmt` this code? Looks like a tab instead of space.

Reply:
done